contributors/design-proposals/storage/local-storage-pv.md

```diff
@@ -761,3 +761,239 @@ The central manager will poll each worker for their discovered volumes and creat
 them.  When a PV is released, then it will send the cleanup request to the worker.
 
 Detailed design TBD
+
+
+### Phase 3: 1.11 Alpha
+
+#### Dynamic Provisioning
+
+The local volume provisioner will be enhanced to support dynamic provisioning of local storage,
+which depends on the newly introduced changes in [Volume Topology-aware Scheduling](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-topology-scheduling.md),
+that is:
+
+1. When the feature gate `DynamicProvisioningScheduling` is enabled,
+the scheduler will verify a pod's volume requirements,
+and set an annotation `annSelectedNode` on PVCs that need provisioning.
+2. New fields `Capacity` will be added in `Status` of `StorageClass` to
+allow storage providers to report capacity limits per topology domain.
+
+The local volume provisioner needs to be updated in several areas to support dynamic provisioning.
+
+##### Backend Storage Management
+
+Mechanisms to manage local storage sources are varied.
+In addition to current `populator`, `deletor` and `discoverer`, a new component named
+`storageManager` will be introduced in the plugin, to manage the backend storage.
+
+According to the input of classes in the configuration, there will be a list of `backends`
+under the manager, to communicate with backend storage. `backend` is a general interface,
+each kind of storage should have its own implementation. storageManager store the backends
+according to storage classes, backends of different classes can be different.
+LVM backend will be introduced in alpha phase.
+
+```
+Backends map[string]backend.DynamicProvisioningBackend
```

Comment:
How about renaming "storageManager" to "dynamicProvisioningManager", then "backends" could be "storageBackends"